### ISSUE TITLE
parse error near `|'

### ISSUE THREAD

**Issue by HaleTom** [softmoth/zsh-vim-mode#3]
```
% source /home/ravi/.config/zsh/.zplugin/plugins/softmoth---zsh-vim-mode/zsh-vim-mode.plugin.zsh
/home/ravi/.config/zsh/.zplugin/plugins/softmoth---zsh-vim-mode/zsh-vim-mode.plugin.zsh:340: parse error near `|'
%
`````

**Comment by softmoth** [comment]
Tom, I'm not able to reproduce this. Can you tell me what ZSH version you're using? Also, would you please try it with a test zshrc to try to identify if it's due to the ZSH version or some conflict with another plugin that is being loaded before?

Try something like
```
zsh --version
mv ~/.zshrc ~/SAVE.zshrc
echo 'source $HOME/.config/zsh/.zplugin/plugins/softmoth---zsh-vim-mode/zsh-vim-mode.plugin.zsh' > ~/.zshrc
```
Then start a new shell and see if you get the error. Also, you can verify the file is current and not corrupted with:
```
md5sum zsh-vim-mode.plugin.zsh
c8585ea6583552dd4bf72bfb8a2371ca  zsh-vim-mode.plugin.zsh
```

Thanks!

**Comment by HaleTom** [comment]
Thanks for your time on this.

I've verified that this doesn't occur when run under `zsh -f`. For now I'm assuming it's a shell option and will update within the next week.

**Comment by softmoth** [comment]
OK, that sounds good. If you do identify a specific cause, let me know.

**Comment by HaleTom** [comment]
The cause is the [zsh-lovers](https://manned.org/zsh-lovers.1) recommended global alias:

```
alias -g C='| wc -l'
```

(The `R` alias will also conflict, on the next line)

These aliases will conflict with [these lines](https://github.com/softmoth/zsh-vim-mode/blob/master/zsh-vim-mode.plugin.zsh#L340)

```
    local -A modes=(
        I  ${vim_mode_indicator_pfx}${MODE_INDICATOR_VIINS}
        C  ${vim_mode_indicator_pfx}${MODE_INDICATOR_VICMD}
        R  ${vim_mode_indicator_pfx}${MODE_INDICATOR_REPLACE}
```

Aliases aren't active when running a script, but this one is sourced.

[Here are some options for disabling aliases](https://unix.stackexchange.com/a/162147/143394).

I'll see if I can bang out a PR.

**Comment by softmoth** [comment]
Great! Thanks for finding this, it's a major issue.

**Comment by softmoth** [comment]
Fixed by #4. Thanks again.
